src/kernel-install/60-ukify.install.in

```diff
@@ -171,6 +171,17 @@ def kernel_cmdline(opts) -> str:
     return ' ' + ' '.join(options)
```

Comment:
The commit message contains several garbage. Please drop them.
Also, we do not use Signed-off-by tag. Please also drop it.